crossdock/client/tchserver/raw.go

```diff
@@ -49,13 +56,42 @@ func runRaw(t crossdock.T, dispatcher yarpc.Dispatcher) {
 	}
 }
 
-func rawCall(dispatcher yarpc.Dispatcher, headers yarpc.Headers, token []byte) ([]byte, yarpc.CallResMeta, error) {
+// remoteTimeout tests if a yarpc client returns a remote timeout error behind
+// the TimeoutError interface when a remote tchannel handler returns a handler
+// timeout.
+func remoteTimeout(t crossdock.T, dispatcher yarpc.Dispatcher) {
+	assert := crossdock.Assert(t)
+	fatals := crossdock.Fatals(t)
+
+	headers := yarpc.NewHeaders()
+	token := random.Bytes(5)
+
+	_, _, err := rawCall(dispatcher, headers, "handlertimeout/raw", token)
+	if skipOnConnRefused(t, err) {
+		return
+	}
+	fatals.Error(err, "expected an error")
```

Comment:
Don't you want to assert that you got an error before you skip based on its value?